pkg/controller/pravega/pravega_controller.go

```diff
@@ -35,8 +35,9 @@ func MakeControllerDeployment(p *api.PravegaCluster) *appsv1.Deployment {
 			Labels:    util.LabelsForController(p),
 		},
 		Spec: appsv1.DeploymentSpec{
-			Replicas: &p.Spec.Pravega.ControllerReplicas,
-			Template: MakeControllerPodTemplate(p),
+			ProgressDeadlineSeconds: func(i int32) *int32 { return &i }(600),
```

Comment:
We could use a local variable instead of this function. That would keep code simple.

Reply:
done